soci/soci_index.go

```diff
@@ -227,17 +233,15 @@ func (b *buildConfig) hasOptimization(o Optimization) bool {
 	return false
 }
 
-// Optimization represents an optional optimization to be applied when building the SOCI index
-type Optimization string
-
 const (
-	// XAttrOptimization optimizes xattrs by disabling them for layers where there are no xattrs or opaque directories
-	XAttrOptimization Optimization = "xattr"
+	// NoneOptimization is a placeholder for any future optimizations.
+	// If future optimizations are added, the below line can be commented/removed.
+	NoneOptimization = "none"
```

Comment:
why do we need this? can we not just leave Optimizations as an empty array?

Reply:
The previous iteration had this entire code block commented out to be example code. We could just make it empty, though.